### ISSUE TITLE
Label of rspec matchers with rails 4.1.6

### ISSUE THREAD

**Issue by thomasstephane** [aasm/aasm#178]
Hi there.

I'm using the state_machine in a rails 4.1.6 app, and the label for matchers is off (was working on rails 4.1.0) despite the test behaving as expected. Any idea?

Test:

```ruby
it { should transition_from(:state_a).to(:state_b).on_event(:change_from_a_to_b) }
```

Result label
```
should #<RSpec::Mocks::Matchers::Receive:0x007fe9f403b778>
```

**Comment by alto** [comment]
@thomasstephane Thanks for the RSpec matcher, it's really awesome! I integrated it – finally – into a new feature branch `rspec_matchers`, in case you wanna have a look. It will be released soon.

**Comment by thomasstephane** [comment]
Nice! Glad I can help.

**Comment by alto** [comment]
Released version `4.5.0` including the new RSpec matchers.
